projects/vdk-heartbeat/src/vdk/internal/heartbeat/config.py

```diff
@@ -224,7 +227,7 @@ def _get_atleast_one_value(self, key1: str, key2: str) -> Optional[str]:
                 f"Countermeasures: Set as required configuration either as environment variable or in the config file."
             )
 
-        return value1 if value1 is not None else value2
+        return value1 or value2 or False
```

Comment:
No, this won't work. Don't you have some unit test failing? If not consider adding unit tests that return empty string for eample

If value1 and value2 are a boolean (with value False) or are a empty string or empty array   

 they all evaluate to "false" in if check which would break the logic.

See https://docs.python.org/2.4/lib/truth.html

The previous logic seemed fine. Why changing it?

Reply:
Ok, I added some tests. I did not find a way to set value1/value2 to boolean values, they are always parsed as strings. But the empty string was a valid case.